config/install/tripal.tripalfield_collection.trpcultivate_experiments.yml

```diff
@@ -776,6 +848,78 @@ fields:
               region: content
               weight: 100
 
+    -   name: researchstudy_projectdescription
+        content_type: research_study
+        label: Project Description
+        type: chado_property_type_default
+        description: Description of a project.
+        cardinality: -1
+        required: false
+        storage_settings:
+            storage_plugin_id: chado_storage
+            storage_plugin_settings:
+                base_table: project
+                prop_table: projectprop
+        settings:
+            termIdSpace: local
+            termAccession: Project Description
+        display:
+          view:
+            default:
+              region: content
+              label: above
+          form:
+            default:
+              region: content
```

Comment:
This is a legacy field that we won't want to include here. Make sure to also remove it's term from the other yaml.